tests/_utils/restart_dm_worker

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# parameter 1: work directory
+# parameter 2: worker-addr port
+# parameter 3: config file for DM-worker
+
+set -eu
+
+workdir=$1
+port=$2
+conf=$3
+
+PWD=$(pwd)
+binary=$PWD/bin/dm-worker.test
+
+echo "[$(date)] <<<<<< START DM-WORKER on port $port, config: $conf >>>>>>"
+cd $workdir
+$binary -test.coverprofile="$TEST_DIR/cov.$TEST_NAME.worker.$port.out" DEVEL \
```

Comment:
It seems this `restart_dm_worker` has quite similar logic to `run_dm_worker` script, is there any purpose to use this one? In addition, the content of coverage file `$TEST_DIR/cov.$TEST_NAME.worker.$port.out` will be truncated here, we'd better to use append stream to this file(`>>`) or choose another file name.

Reply:
remove `mkdir` and `ln` opeations, maybe we also add other operation in it later, how do you think about it?

Reply:
`mkdir -p` is reentrant, and we can also update `ln` to `[ -f $workdir/bin/mydumper ] || ln -s $PWD/bin/mydumper $workdir/bin/mydumper` to make it reentrant